wasmtime-jit fails to built when compiled standalone

The compilation succeeds if `cargo build` is executed from the repository root, but fails if `cargo build` is executed from `wasmtime-jit`. Tested on Arch Linux, nightly. (`feature(alloc)` may not be used on stable)

```
$ cd wasmtime-jit
$ cargo build 
...
   Compiling wasmtime-environ v0.1.0 (/home/marcin/proj/wasmtime/wasmtime-environ)
error[E0432]: unresolved import `std::ops`
  --> wasmtime-environ/src/compilation.rs:10:10
   |
10 | use std::ops::Range;
   |          ^^^ help: a similar path exists: `rayon::std::ops`

warning: trait objects without an explicit `dyn` are deprecated
  --> wasmtime-environ/src/cranelift.rs:92:51
   |
92 | fn get_address_transform(context: &Context, isa: &isa::TargetIsa) -> Vec<InstructionAddressMap> {
   |                                                   ^^^^^^^^^^^^^^ help: use `dyn`: `dyn isa::TargetIsa`
   |
   = note: #[warn(bare_trait_objects)] on by default

error[E0282]: type annotations needed
  --> wasmtime-environ/src/compilation.rs:47:27
   |
47 |                     body: buffer[range].to_vec(),
   |                           ^^^^^^^^^^^^^ cannot infer type
   |
   = note: type must be known at this point

error: aborting due to 2 previous errors

Some errors have detailed explanations: E0282, E0432.
For more information about an error, try `rustc --explain E0282`.
error: Could not compile `wasmtime-environ`.

To learn more, run the command again with --verbose.
```

--- a/wasmtime-debug/Cargo.toml
+++ b/wasmtime-debug/Cargo.toml
@@ -25,7 +25,7 @@ failure_derive = { version = "0.1.3", default-features = false }
 
 [features]
 default = ["std"]
-std = ["cranelift-codegen/std", "cranelift-wasm/std"]
+std = ["cranelift-codegen/std", "cranelift-wasm/std", "wasmtime-environ/std"]
 core = ["cranelift-codegen/core", "cranelift-wasm/core"]
 
 [badges]

--- a/wasmtime-jit/Cargo.toml
+++ b/wasmtime-jit/Cargo.toml
@@ -28,7 +28,7 @@ wasmparser = "0.32.1"
 
 [features]
 default = ["std"]
-std = ["cranelift-codegen/std", "cranelift-wasm/std"]
+std = ["cranelift-codegen/std", "cranelift-wasm/std", "wasmtime-environ/std"]
 core = ["hashbrown/nightly", "cranelift-codegen/core", "cranelift-wasm/core", "wasmtime-environ/core"]
 lightbeam = ["wasmtime-environ/lightbeam"]
 

